packages/node_modules/@ciscospark/plugin-lyra/src/spaces.js

```diff
@@ -0,0 +1,393 @@
+/**!
+ *
+ * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import {SparkPlugin} from '@ciscospark/spark-core';
+import querystring from 'querystring';
+import {base64} from '@ciscospark/common';
+
+/**
+ * @class
+ * @extends {Lyra}
+ * @memberof Lyra
+ */
+const Spaces = SparkPlugin.extend({
```

Comment:
If I'm reading this right, I think this should be split into two plugins:
- space
- device

There shouldn't be any need to include "Space" in method names since the methods are attached to the space plugin.